### ISSUE TITLE
Support OwnPush

### ISSUE THREAD

**Issue by engelgabriel** [raix/push#191]
We at Rocket.Chat are investigating the integration of OwnPush's end-to-end encrypted push notifications into our project. 

See https://github.com/RocketChat/Rocket.Chat/issues/2124

It would be great if this can be pushed upstream to raix:push if possible - and the greater Meteor development community can benefit from this work. @raix, what do you think? is this a possibility?

We realize that OwnPush is currently Android only and support may require the addition of native binding code, but they are willing to collaborate on the effort.

**Comment by raix** [comment]
It should be possible to integrate, is it on cordova?

**Comment by engelgabriel** [comment]
I don't think so, not yet. But if you are willing to integrate into your this package, they are willing to build the Cordova plugin.

**Comment by jerdog** [comment]
We at OwnPush are assisting Rocket.Chat's team to use this with Cordova and can certainly assist your team in this endeavor as well.
